bin/smoke

```diff
@@ -216,8 +224,7 @@ end
 
 begin
   @options, @args = parse_options
-  application = @args[0]
-  validate_application(application)
+  application = @args[0].split(/ /) # quick hack, does not consider blanks in file names
```

Comment:
I was thinking you could have a generic `command` file, and a `test-name.command` file for an individual test that would override it.

The command would be concatenated with the `args` file.